src/ResourceManager/Resources/Commands.Resources.UnitTests/RoleAssignmentUnitTests.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Microsoft.Azure.Commands.Resources.Models.Authorization;
+using Microsoft.Azure.Commands.Resources;
+using System.Collections.Generic;
+
+namespace Commands.Resources.UnitTests
+{
+    [TestClass]
```

Comment:
MSTest tests are not allowed. Please make these xunit tests, and mark them as check-in tests